guides/release/templates/actions.md

```diff
@@ -22,21 +22,27 @@ export default class Post extends Component {
 ```
 
 You can then add this action to an element using the
-[`{{action}}`](https://api.emberjs.com/ember/release/classes/Ember.Templates.helpers/methods/action?anchor=action)
+[`{{on}} (link tbd)`](https://api.emberjs.com/ember/release/classes/Ember.Templates.helpers/methods/action?anchor=action)
```

Comment:
This may not be specific to this PR, but this seems like the best place to bring it up…

There's a conceptual leap that happens here, imo, that has been historically obscured by the duplication of the word `action`. Switching to `{{on}}`, thankfully, brings it to light. Namely, why does `toggleBody` need to be decorated with `@action`? What does that decoration actually _do_? For example:

```javascript
export default class MyButtonComponent extends Component {
  fire() {
    alert("fire!");
  }
}
```
and

```handlebars
<button {{on "click" this.fire}}>
  Click me!
</button>
```
works as expected—the `{{on}}` helper is enough to create interactivity in this example, without having to use `@action`.

Reply:
ah, yeah. `@action` is needed when you want `this` to be bound -- I'll be sure that is mentioned somewhere

Reply:
yes. I think I understand this now!

Reply:
(but because of your and @locks interventions! which means I think I can try to help clarify the guide)

Reply:
tl;dr: binding is very confusing to me.

Reply:
I'll try to find an article that can explain it nicely. It's pretty fundamental to javascript, so It's pretty important we find a good and clear explanation

Reply:
@muziejus, thoughts on this article: https://yehudakatz.com/2011/08/11/understanding-javascript-function-invocation-and-this/

Reply:
Per our chat on Discord, this is the line that trips me up:

> Actions are methods that have been decorated with the @action decorator in the context of the template:

I would propose something like this (my terminology might be inexact):

> Actions are methods that have been decorated with the @action decorator in the context of the template. The decorator binds `this` to the `Post` component:

Then, perhaps, after the code, where the PR has:

> You can then add this action to an element using the [`{{on}} (link tbd)`]

Instead it can be:

> In binding the context, `@action` lets `toggleBody()` access the component's other properties when invoked from the template with `this`. Without the decorator, `this.isShowingBody` in `toggleBody()` is undefined, as in that case `this` refers not to the `Post` component, but to the `window` object. 

> You can trigger the method `toggleBody()` as a callback on a DOM event using the [`{{on}} (link tbd)`]...



This is rather verbose, but, to me at least, actually explains *why*, within an *Ember* context, you need to use the decorator.

Reply:
I've updated the text for this, based on your suggestions, feel free to comment on the new stuff :)